src/lib.rs

```diff
@@ -56,7 +33,7 @@ mod config;
 /// All internal error handling logic
 mod error;
 /// Networking modules
-mod net;
+pub mod net;
```

Comment:
do we want this to be public? I thought we re-exported all types in `lib.rs`

Reply:
For some reason, it threw an error. I'll see if I can figure out how to do it without making that public.

Reply:
Huh, works without it now. Ok then.